wrims-gui/build.gradle

```diff
@@ -0,0 +1,60 @@
+plugins {
+    id 'library.deps-conventions'
+    id 'library.java-conventions'
+}
+
+ext {
+    wrimsGuiFolder = "wrims2_gui_x64_${version}"
+}
+
+configurations.all {
+    exclude group: "javax.media", module: "jai_core"
+    exclude group: "javax.media", module: "jai_codec"
+    exclude group: "com.sun.media", module: "jai_imageio"
+}
+
+// Task to unzip wrims_gui_template.zip
+task unzipWrimsGuiTemplate(type: Copy) {
+    from(zipTree('J:/WRIMS/wrims_gui_template.zip'))
+    into("${buildDir}/${wrimsGuiFolder}")
+}
+
+// Task that removes jars containing "hecdssvue", "jdiagram", or "wrimsv2" from the ${buildDir}/${wrimsGuiFolder}/dropins folder
+task removeJarsFromDropins(type: Delete) {
+    delete fileTree(dir: "${buildDir}/${wrimsGuiFolder}/dropins", include: ['*hecdssvue*.jar', '*jdiagram*.jar', '*wrimsv2*.jar', '*xtext*.jar'])
+}
+
+removeJarsFromDropins.dependsOn(unzipWrimsGuiTemplate)
+
+// Task to copy JARs to dropins folder
+task copyJarsToDropins(type: Copy) {
+    from(fileTree(dir: '../third-party/build/libs', include: '*.jar'))
```

Comment:
This should grab the output of the jar task for the project rather than rely on glob wildcarding. Sources, javadoc, and test fixture jars would all match these patterns in the libs folder.

Reply:
Josh updated